tock/hours/tests/test_models.py

```diff
@@ -185,6 +196,24 @@ def test_timecard_max_target_hours(self):
 
         self.assertEqual(self.timecard.target_hours, self.timecard.max_target_hours())
 
+    def test_timecard_saves_user_organization(self):
```

Comment:
Related to above, let's tweak these tests to confirm both the application of the org/unit attributes but that they're only effect on creation not every save.